src/Uri.php

```diff
@@ -65,7 +65,7 @@ class Uri implements UriInterface
     /**
      * @param string $uri URI to parse
      */
-    public function __construct($uri = '')
+    public function __construct(string $uri = '')
     {
         // weak type check to also accept null until we can add scalar type hints
         if ($uri != '') {
```

Comment:
please change it to `!==` and remove the comment above

Reply:
Done.. resolving.